assets/js/components/dashboard-sharing/UserRoleSelect.js

```diff
@@ -50,9 +50,13 @@ const { useSelect, useDispatch } = Data;
 const ALL_CHIP_ID = 'all';
 const ALL_CHIP_DISPLAY_NAME = __( 'All', 'google-site-kit' );
 
-export default function UserRoleSelect( { moduleSlug, isLocked = false } ) {
+export default function UserRoleSelect( {
+	moduleSlug,
+	isLocked = false,
+	moduleRef,
```

Comment:
Can we use `forwardRef` instead of passing as a regular prop? I think it would be more consistent with our codebase. See:
https://github.com/google/site-kit-wp/blob/d7d12170ef2854d55276eb268a3682efc68191de/assets/js/components/Menu.js#L42